post-scan-actions/azure-python-slack-notification/slackNotification/ScanResultHandler/handler.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import os
+import json
+import urllib3
+http = urllib3.PoolManager()
+import textwrap
+
+import azure.functions as func
+from azure.identity import DefaultAzureCredential
+
+# Hide verbose HTTP logging
+logging.getLogger('azure.core.pipeline.policies.http_logging_policy').setLevel(logging.WARNING)
+
+
+def main(message: func.ServiceBusMessage):
+    # set slack variables
+    url = os.environ["SLACK_URL_WEBHOOK"]
+    account = os.environ["SUB_NAME"]
+    channel = os.environ["SLACK_CHANNEL"]
+
+    # Log the Service Bus Message as plaintext
+    message_body = message.get_body().decode("utf-8")
+    logging.info('Python ServiceBus topic trigger processed message.')
+    logging.info(f'Message Body: {message_body}')
+
+    message = json.loads(message_body)
+    findings = message['scanning_result'].get('Findings')
+
+    if findings:         
+        malwares = []
+        types = []
+        for finding in findings:
+            malwares.append(finding.get('malware'))
+            types.append(finding.get('type'))
+
+        body_text = textwrap.dedent('''
+        WARNING
+        Azure Subscription: {account_id}
```

Comment:
I just realize that the subscription name is used in the notification message. Supposedly it should be the subscription of the object's storage account, right? Since one storage stack can monitor storages in different subscriptions and they can be different from the plugin's subscription, it shouldn't be retrieved by ARM function or an input of the ARM deployment. Instead of providing subscription name, I would prefer providing tenant display name (`[tenant().displayName]`). Supposedly storage account name in `file_url` would be enough for them to find the storage account on Azure portal. Any thought?

Reply:
I was trying to just make it obvious in the slack message from a glance. Also trying to mimic the aws slack notification format with the aws account #.
Would this be a change to make in the template.json file?

for example:
"name": "SUB_NAME",
"value": "[tenant().displayName]"

Reply:
Yes, and I suggest to use another configuration name, like `TENANT_NAME`, to match the value we use.

Reply:
Updated to be tenant_name && tenant().[displayName].
Changed in handler.py variable account to reflect change

Reply:
```suggestion
        Azure Tenant: {account_id}
```

Reply:
Fixed